package.json

```diff
@@ -93,7 +93,7 @@
     "@babel/plugin-syntax-dynamic-import": "7.8.3",
     "@babel/plugin-transform-modules-commonjs": "7.9.0",
     "@babel/polyfill": "7.8.3",
-    "@babel/preset-env": "7.8.3",
+    "@babel/preset-env": "7.8.7",
```

Comment:
Why?

Reply:
my build was le-broken (was receiving one of these errs: https://github.com/babel/babel/issues/12019)